rust/processor/src/config.rs

```diff
@@ -65,6 +69,11 @@ impl IndexerGrpcProcessorConfig {
     pub const fn default_pb_channel_txn_chunk_size() -> usize {
         100_000
     }
+
+    /// Default timeout for grpc response item in seconds. Defaults to 300 seconds.
+    pub const fn default_grpc_response_item_timeout_in_secs() -> u64 {
+        60 * 5
```

Comment:
can we decrease this timeout any more so the processors reconnect faster? message size limit on grpc side is 15MB

Reply:
true; this number is from previous setting.

15MB / 10mbps = 15 seconds

Reply:
updated.

Reply:
15 is too aggressive; we want to relax this condition considering network contention may happen.